server/src/main/java/umm3601/user/UserController.java

```diff
@@ -99,8 +99,14 @@ public void getUsers(Context ctx) {
       filters.add(eq(ROLE_KEY, ctx.queryParam(ROLE_KEY)));
     }
 
-    String sortBy = ctx.queryParam("sortby", "name"); //Sort by sort query param, default is name
-    String sortOrder = ctx.queryParam("sortorder", "asc");
+    String sortBy = ctx.queryParam("sortby"); //Sort by sort query param, default is name
+    if (sortBy == null) {
+      sortBy = "name";
+    }
+    String sortOrder = ctx.queryParam("sortorder");
+    if (sortOrder == null) {
+      sortOrder = "asc";
+    }
```

Comment:
It's too bad they removed the default value from `ctx.queryParam`. This really highlights how clunky Java can be! In Kotlin this could simply be 

```kotlin
val sortBy = ctx.queryParam("sortby") :? "name"
```

Though actually as I just found, in Java > 9 there is a way make this a bit nicer using [Objects#requireNonNullElse](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/util/Objects.html#requireNonNullElse(T,T)):
```suggestion
    String sortBy = Objects.requireNonNullElse(ctx.queryParam("sortby"), "name"); //Sort by sort query param, default is name
    String sortOrder = Objects.requireNonNullElse(ctx.queryParam("sortorder"), "asc");
```

Reply:
Nice find! That code was pretty unattractive and I was actually thinking about writing a little helper function that basically does what `requireNonNullElse` presumably does.